packages/cli/src/cmds/assert/assertion.ts

```diff
@@ -0,0 +1,6 @@
+import { Scope } from './types';
+
+export default class Assertion {
+  constructor(public scope: Scope, public where: string, public assert: string) {
```

Comment:
What does public mean on a parameter?

Reply:
The `public` keyword instructs the TypeScript compiler that it’s OK to access the property from outside the class.